test/core/parser/parse_test.py

```diff
@@ -48,3 +49,15 @@ def test__parser__parse_error():
         "after <WordSegment: ([L:  1, P:  1]) 'SELECT'>. "
         "Found nothing."
     ) in parsed.tree.stringify()
+
+
+def test_parse_jinja_macro_exclude():
+    """Test parsing when excluding macros with unknown tags."""
```

Comment:
Can we be a little more explicit here, just so it's really obvious what this test is about?

```suggestion
    """Test parsing when excluding macros with unknown tags.

    This test case has a file which defines the unknown tag `materialization` which would
    cause a templating error if not excluded. By ignoring that folder we can ensure there
    are no errors.
    """
```